.github/workflows/main.yaml

```
@@ -0,0 +1,48 @@
+name: Main
+
+on:
+    push:
+        branches:
+            - main
+            - master
+    pull_request:
+        branches:
+            - main
+            - master
+
+jobs:
+    jest:
+        name: Jest
+        runs-on: ${{ matrix.os }}
+
+        strategy:
+            matrix:
+                os: [ubuntu-latest]
+                node: [14]
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@master
```

Use versioned actions:

```suggestion
              uses: actions/checkout@2
```